parastell.py

```diff
@@ -780,23 +735,32 @@ def parastell(
     # Conditionally build graveyard volume
     if export_dict['graveyard']:
         # Extract maximum offset
-        offset = 2*max(max(offset_mat))
+        offset = max(max(offset_mat))
         # Build graveyard
         components = graveyard(vmec, offset, components, logger)
 
     # Conditionally initialize Cubit
     if export_dict['h5m_export'] == 'Cubit' or magnets is not None:
+        # Retrieve Cubit module directory
+        cubit_dir = os.path.dirname(inspect.getfile(cubit))
+        # Append plugins directory to Cubit module directory
+        cubit_dir = cubit_dir + '/plugins/'
+        # Initialize Cubit
         cubit.init([
             'cubit',
             '-nojournal',
             '-nographics',
             '-information', 'off',
-            '-warning', 'off'
+            '-warning', 'off',
+            '-commandplugindir',
+            cubit_dir
         ])
```

Comment:
this should probably happen as early as possible so that you can fail early if cubit is missing or mis-configured